mote/soke.py

```diff
@@ -105,9 +107,14 @@ def run():
                 except:
                     pass
 
+    # fetch latest meetings using datagrepper for the past 24 hours
+    latest_meetings = get_latest_meetings()
```

Comment:
This needs exception handling around it, in case datagrepper is down.  If datagrepper dies, perhaps just let ``latest_meetings = []`` (graceful failure).